decentraland/unity-explorer
# Remove and update packages

## What does this PR change?



This change removes some packages that are unused and safe to remove and upgrades others that I believe are safe to update. Some packages removed from the manifest are still used if they are dependencies of other packages.

Fewer packages means less work for Unity editor when loading or reloading the project, leading to slightly faster iteration times. Build time and size should also go down at least a little bit.

## How to test the changes?



1. Launch the explorer
2. Play normally
3. The game should behave exactly the same as without this change

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
